scripts/benchmarking.py

```diff
@@ -105,7 +105,7 @@ def recall(predicted, relevant, eval_at):
             {'n_dim': D},
         ),
         ('qdrant', {'n_dim': D, 'scroll_batch_size': 8}),
-        ('weaviate', {'n_dim': D}),
+        ('weaviate', {'n_dim': D, 'distance': 'l2-squared'}),
```

Comment:
this changes should be in another PR

Reply:
That makes sense, please see [this](https://github.com/jina-ai/docarray/pull/487#issuecomment-1214382690) and let me know if that works for you.